app/controllers/admin/schools/partnerships_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Admin
+  module Schools
+    class PartnershipsController < AdminController
+      layout "full"
+
+      before_action :setup_school_page
+      before_action :setup_navigation_items
```

Comment:
Is there any advantage in using `before_action` when there's only one method? :sweat_smile: